charts/spire/examples/production/your-values.yaml

```diff
@@ -0,0 +1,20 @@
+# These values should be in your own configuration values.yaml
+# You need to override them as appropriate for your deployment
+
+global:
+  spire:
+    clusterName: changeme
+    trustDomain: changeme.yourdomain
+
+spire-server:
+  jwtIssuer: oidc-discovery.changeme.yourdomain
+  ca_subject:
+    country: NL
+    organization: ChangeMe
+    common_name: chageme.yourdomain
+
+# If using spiffe-oidc-discovery-provider
```

Comment:
I would prefer that we have the oidc-discovery-provider disabled by default for a generalized production example.